### ISSUE TITLE
Add Covid Detection 

### ISSUE THREAD

**Issue by SayanSaha01** [mohit200008/medi-Care#54]
I wanted to add the Covid Detection model which predicts whether a person is covid +ve or not based upon chest X-Ray

**Comment by mohit200008** [comment]
Yes you can work on this issue @SayanSaha01 .  Currently  the backend is on flask but it has to be migrated to django

**Comment by mohit200008** [comment]
@SayanSaha01  you working on this issue ?
